src/components/Book.module.css

```diff
@@ -0,0 +1,69 @@
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 1.5em 1.5em;
+  border-radius: 4px;
+  border: solid 1px #e7e5e5;
+  margin: 1em 0;
+  background-color: #fff;
+}
+
+.book {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+}
```

Comment:
- [ ] In order to align with CSS best practices of keeping your code DRY I'd suggest moving the `display: flex` property into a separate class (along with other properties if you wish) and using that class name on the different elements that need the styling instead of repeating the property multiple times in your styles file.
- [ ] You can also use the same principle for other properties that are being repeated in your styles.